### ISSUE TITLE
why the results are not stable？

### ISSUE THREAD

**Issue by wuqiangch** [luoyetx/deep-landmark#7]
Without changing other paramers, trainging again can get better (maybe worse) results?

**Comment by luoyetx** [comment]
Because of the network initialization, there's no guarantee that the loss will be stable or able to reach a point gives you a low loss. You should consider to restart the training or tuning some parameters if the loss is not what you want.

**Comment by wuqiangch** [comment]
If I get more data and finetune the models . Maybe I can get better results, because  I have get a  better network initialization.

**Comment by luoyetx** [comment]
more data can help training the network but the network initialization part has nothing to do with your data. Actually, the default weight initialization method is ok for most situation and the loss is likely to converge and become small with training going on.
